aiida_mlip/calculations/singlepoint.py

```diff
@@ -152,12 +95,12 @@ def validate_inputs(
         """
         # Wrapping processes may choose to exclude certain input ports
         # If the ports have been excluded, skip the validation.
-        if "structure" not in port_namespace:
+        if "structure" not in port_namespace and "config" not in port_namespace:
             raise ValueError("'Structure' namespaces is required.")
 
         if "input_filename" in inputs:
             if not inputs["input_filename"].value.endswith(".xyz"):
-                raise ValueError("The parameter 'input_filename' must end with '.cif'")
+                raise ValueError("The parameter 'input_filename' must end with '.xyz'")
```

Comment:
it is because of ase, I just wanted to catch the error at an aiida level so I can do stuff with it (like I am calling it Inputvalidationerror, rather than a generic valueerror).
but can remove it if it's redundant